app/Mail/ScreenCandidatesPrompt.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace App\Mail;
+
+use App\Models\JobPoster;
+use App\Models\Lookup\CitizenshipDeclaration;
+use App\Models\Lookup\VeteranStatus;
+use App\Models\User;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Config;
+use Illuminate\Support\Facades\Lang;
+use Illuminate\Support\Facades\Log;
+use Mcamara\LaravelLocalization\Facades\LaravelLocalization;
+
+class ScreenCandidatesPrompt extends Mailable implements ShouldQueue
+{
+    use Queueable, SerializesModels;
+
+    /**
+     * The Job Poster instance.
+     *
+     * @var JobPoster
+     */
+    public $job;
+
+    /**
+     * Create a new message instance.
+     *
+     * @return void
+     */
+    public function __construct(JobPoster $job)
+    {
+        $this->job = $job;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        // Get emails of all Hr Advisors who have claimed job.
+        $hr_advisors_emails = [];
+        $hr_advisors = $this->job->hr_advisors;
+
+        foreach ($hr_advisors as $advisor) {
+            $email = User::where('id', $advisor->user_id)->first()->email;
+            array_push($hr_advisors_emails, $email);
+        }
+
+        // Get date 2 weeks after job poster closes.
+        $dateFormat = Config::get('app.date_format');
+        $locale = App::getLocale();
+        $timestamp_in_two_weeks = strtotime('+2 weeks', $this->job->close_date_time->timestamp);
+        $date_in_two_weeks = date($dateFormat[$locale], $timestamp_in_two_weeks);
+
```

Comment:
Not sure if this really affects anything, but we're using a date library in most other files ([Jenssegers Date](https://github.com/jenssegers/date)) for datetime manipulation.

The date formatted in this way will also be in UTC (again, don't know if it matters since you aren't displaying time) but it might be +/-1 day from EST/EDT/wherever.

I had to do some finagling in the `JobPoster::applyBy()` method:
```php
        $localCloseDate = new Date($this->close_date_time); // This initializes the date object in UTC time.
        $localCloseDate->setTimezone(new \DateTimeZone(self::TIMEZONE)); // Then set the time zone for display.
        $displayDate = [
            'date' => $localCloseDate->format(self::DATE_FORMAT[App::getLocale()]),
            'time' => $localCloseDate->format(self::TIME_FORMAT[App::getLocale()])
        ];
```